platform/linux-generic/odp_queue_basic.c

```diff
@@ -949,6 +950,20 @@ static int queue_api_enq_multi(odp_queue_t handle,
 				      (odp_buffer_hdr_t **)(uintptr_t)ev, num);
 }
 
+static void queue_timer_add(odp_queue_t handle)
+{
+	queue_entry_t *queue = qentry_from_handle(handle);
+
+	odp_atomic_inc_u64(&queue->s.timers);
+}
+
+static void queue_timer_rem(odp_queue_t handle)
+{
+	queue_entry_t *queue = qentry_from_handle(handle);
+
+	odp_atomic_dec_u64(&queue->s.timers);
+}
```

Comment:
these 2 functions repeated in all schedulers. how about to put them to .h ?

Reply:
As the queue implementations should be independent I wouldn't like to add a common header file for such trivial functions.

Reply:
Agree with @matiaselo on this. I'm OK with these in each scheduler.